pkg/spdk/engine.go

```diff
@@ -796,9 +805,13 @@ func (e *Engine) ValidateAndUpdate(spdkClient *spdkclient.Client) (err error) {
 		}
 	}
 
-	e.log.UpdateLogger(logrus.Fields{
-		"replicaStatusMap": e.ReplicaStatusMap,
-	})
+	replicaStatusMap := make(map[string]*EngineReplicaStatus)
```

Comment:
NIT: It's better to add a flag like `replicaStatusMapChanged` to control if this logger field `replicaStatusMap` is required as this will be frequently triggered.

Reply:
Updated. Thank you.